main_test.go

```diff
@@ -109,20 +114,46 @@ func (suite *EndToEndTestSuite) TestIndexPerson() {
 
 	err = json.NewDecoder(resp.Body).Decode(&iResp)
 	if err != nil {
-		suite.Fail("Unable to decode JSON response", resp.Body)
+		suite.Fail("Unable to decode JSON index response", resp.Body)
 	}
 
 	expectedResp := response.IndexResponse{
 		Results: []elasticsearch.IndexResult{
 			{
 				Id:         3,
 				StatusCode: 201,
-				Message:    "Index created",
+				Message:    "Document created",
 			},
 		},
 	}
 
 	suite.Equal(expectedResp, iResp, "Unexpected index result")
+
+	time.Sleep(time.Second * 2)
```

Comment:
Yeah good point. Without the sleep the search query doesn't find the document. I guess ES needs a moment to update the index and/or its internal cache?

Reply:
This would probably be better if the fetch from ES was wrapped in a loop and retried on shorter increments. 2 seconds is a long time for a test to wait.

Reply:
True. It only needed 0.6s